chart/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.8"
-version: "1.1.3"
+appVersion: "1.9"
```

Let's go for 2.0 since this is the largest update this project has had since being created 🙂

```suggestion
appVersion: "2.0"
```